rust/processor/src/db/postgres/migrations/2024-06-27-213336_add_v1_migration_view/up.sql

```diff
@@ -0,0 +1,18 @@
+-- Your SQL goes here
+CREATE OR REPLACE VIEW legacy_migration_v1.current_collection_datas AS
```

Comment:
Do we need to add any indexes?

Reply:
oh i was looking at local db and that's why there was none. will add!

Reply:
Yeah double check because the indexes might already exist in the v2 table

Reply:
if there is a index, should we put this in a separate migration to avoid the issue we had last time?

Reply:
will probably add and comment out just in case and make a note to do it outside of diesel migration